### PR TITLE
WIP: Fixing the formatting

### PR DESCRIPTION
**Pull request by VoyTechnology**
Why do we have so many pages...

This change essentially contains reformatting of all the code blocks as well as capping the line length to 80 (because some paragraphs were just one one line, so I decided to have a 80 line cap.

### CODE REVIEW

File: docs/dovecot.md [hunk 1]
```diff
@@ -1,30 +1,34 @@
...
+```
+mail_location = maildir:~/Maildir:INDEX=/var/mail/indexes/%u
 
-	mail_location = maildir:~/Maildir:INDEX=/var/mail/indexes/%u
-
-
-	[root@morpheus /]# ls -ld /var/mail/indexes
-	drwxrwsrwt 58 root mail 4.0K 2009-07-28 14:11 /var/mail/indexes
 
+[root@morpheus /]# ls -ld /var/mail/indexes
```
Comment (butlerx) [inline docs/dovecot.md]: paphos

File: docs/dracaccess.md [hunk 2]
```diff
@@ -51,24 +50,25 @@ Note that carbon is on .2 because at some point there was a gateway on .1, or cr
  | .123  _______________
                                    -------------------=| pike (IP-KVM) |
                                                         ---------------
+*Edit(voy): How am I even supposed to figure it out?...*
```
Comment (butlerx) [inline docs/dracaccess.md]:
```
  |                                             .2    ______________
__|______                        -------------------=| carbon DRAC  |
|       |  .8               _____|_                  ~~~~~~~~~~~~~~~
|sprout |=-----------------|  hub  |
|_______|                   ~~~~~|~             .3    _______________
                                 -------------------=| deathray DRAC |
                                 |                    ~~~~~~~~~~~~~~~
                                 |
                                 |              .14   _______________
                                 -------------------=|  murphy ALOM  |
                                 |                    ~~~~~~~~~~~~~~~
                                 |
                                 |              .23   _______________
                                 -------------------=|   cynic RSC   |
                                 |                    ~~~~~~~~~~~~~~~
                                 |
                                 |              .123  _______________
                                 -------------------=| pike (IP-KVM) |
                                                      ---------------
```

File: docs/legacy/machines/murphy.md [hunk 3]
```diff
@@ -4,140 +4,142 @@
...
-
-	[root@murphy ~]# ls /etc/exim    
-	lrwxrwxrwx   1 root     root          17 Jan  4 05:32 /etc/exim -> /opt/csw/etc/exim
-	[root@murphy ~]# ls /var/log/exim
-	lrwxrwxrwx   1 root     root          21 Jan  4 05:43 /var/log/exim -> /opt/csw/var/log/exim
-
-
-All of the exim stuff is in /opt/csw, those symlinks are there for convienience. The installed version is over a year old, but there's no major fixes since then.
+```
+[root@murphy ~]# ls /etc/exim
```
Comment (butlerx) [inline docs/legacy/machines/murphy.md]: paphos again

File: docs/carbonanddeathray.md [hunk 4]
```diff
@@ -1,18 +1,35 @@
...
-    LDAP 1      mysql     DNS 1       DNS 2        Sec web
-              pgsql     Mail 2      Mail 1
-                        Boards      Squid cache?
 
-*Note - Sec(ondary) web isn't a full fledged copy of what we're running on Murphy right now. Rather, it would be a place for the random little tidbits that really shouldn't go down when the important machines go down (ie. the kind of crap we're running on obelisk right now - these docs, anyterm, irc proxy, etc). Obelisk is a heap of junk compared to cynic.*
+| Minerva | Murphy | Carbon | Deathray | Cynic |
+| ------- | ------ | ------ | -------- | ----- |
+| Login   | P web  | LDAP 2 | LDAP 3   | DNS 3 |
+| LDAP 1  | MySQL  | DNS 1  | DNS 2    | S web |
+|					| PgSQL  | Mail 2 | Mail 1   |       |
```
Comment (butlerx) [inline docs/carbonanddeathray.md]: ive seen it renders fine just wondering about the odd spacing in plain txt

File: docs/2011-to-do.md [hunk 5]
```diff
@@ -1,12 +1,10 @@
-- Stop azazel turning itself off every 60 days.
-
```
Comment (VoyTechnology) [inline docs/2011-to-do.md]: Doesn't seem to be a problem.

File: docs/committeechangeover.md [hunk 6]
```diff
@@ -1,132 +1,133 @@
...
+
+Enter who updated this user (give Unix username)
+[root] >> atlas
+
+User converted: $new_committee_member -> committe
+Account converted: $new_committee_member -> committe
+Subscribed: $new_committee_member@redbrick.dcu.ie
+```
+
+*Edit(voy): I presumed this is what it was. The layout was a bit messed up.*
```
Comment (VoyTechnology) [inline docs/committeechangeover.md]: Acknowledged.

File: docs/carbonanddeathray.md (hunk 4 above)
Comment (VoyTechnology) [inline docs/carbonanddeathray.md]: Done.

File: docs/committeechangeover.md [hunk 7]
```diff
@@ -1,132 +1,133 @@
...
-memberUid: $new_webmaster**\\
-\\
-modifying entry “cn=helpdesk,ou=groups,o=redbrick”\\
-''
-\\
-(Bold text is what the user types.)
-
-Add the webmaster to the webmaster mailing list (webmaster `<at>` lists.redbrick.dcu.ie) at http://lists.redbrick.dcu.ie/mailman/admin/webmaster/
+```text
+root@carbon# ldapmodify -D cn=root,ou=ldap,o=redbrick -x -y /etc/ldap.secret
```
Comment (VoyTechnology) [inline docs/committeechangeover.md]: Acknowledged.

File: docs/committeechangeover.md [hunk 8]
```diff
@@ -1,132 +1,133 @@
...
+
+*Edit(voy): I presumed this is what it was. The layout was a bit messed up.*
+
+This changes the `gidNumber` of the user in `LDAP` to `100` (the `committee`
+group). It also subscribes the user to the committee mailing list. Do this for
+each of the new committee members.
 
 ### IRC
 
 #### #rbcommittee
```
Comment (VoyTechnology) [inline docs/committeechangeover.md]: Acknowledged.

File: docs/committeechangeover.md [hunk 9]
```diff
@@ -1,132 +1,133 @@
...
+	add: memberUid
+	memberUid: $new_webmaster
 
-### Helpdesk
+	modifying entry “cn=helpdesk,ou=groups,o=redbrick”
+```
 
-Add each member to the helpdesk mailing lists (TODO, list all lists).
+Add the webmaster to the webmaster mailing list
+(webmaster@lists.redbrick.dcu.ie) at
```
Comment (VoyTechnology) [inline docs/committeechangeover.md]: I presume so.

File: docs/committeechangeover.md [hunk 10]
```diff
@@ -1,132 +1,133 @@
...
+http://lists.redbrick.dcu.ie/mailman/admin/webmaster/
 
-Add each member to the LDAP helpdesk group:
+### Helpdesk
+Add each member to the `helpdesk` mailing lists (TODO, list all lists).
```
Comment (VoyTechnology) [inline docs/committeechangeover.md]: Not part of this PR.

File: docs/committeechangeover.md [hunk 11]
```diff
@@ -1,132 +1,133 @@
...
-add: memberUid\\
-memberUid: $new_helpdesk_member1\\
-memberUid: $new_helpdesk_member2\\
-memberUid: $new_helpdesk_member3**\\
-modifying entry "cn=helpdesk,ou=groups,o=redbrick"''
+Add each member to the LDAP helpdesk group:
 
-(Bold text is what the user types.)
+```text
+root@deathray:~# ldapmodify -D cn=root,ou=ldap,o=redbrick -x -y /etc/ldap.secret
```
Comment (VoyTechnology) [inline docs/committeechangeover.md]: Acknowledged.

File: docs/committeechangeover.md [hunk 12]
```diff
@@ -1,132 +1,133 @@
...
-''root@deathray:~# **ldapmodify -D cn=root,ou=ldap,o=redbrick -x -y /etc/ldap.secret\\
-dn: cn=root,ou=groups,o=redbrick\\
-changetype: modify\\
-add: memberUid\\
-memberUid: $new_admin_member1\\
-memberUid: $new_admin_member2\\
-memberUid: $new_admin_member3**\\
-modifying entry "cn=root,ou=groups,o=redbrick"''
+* Add each member to the admin mailing lists (rb-admins and elected-admins).
+* Add each member to the LDAP root group:
```
Comment (VoyTechnology) [inline docs/committeechangeover.md]: Nope - they won't (unless `mkdocs` renders the it differently than readthedocs does)

File: docs/committeechangeover.md [hunk 13]
```diff
@@ -1,132 +1,133 @@
...
 
-(Bold text is what the user types.)
+```text
```
Comment (VoyTechnology) [inline docs/committeechangeover.md]: Kinda. Because there is `#` in there, the command looks like a comment. So its not ideal to use `bash`. With `text` we loose the syntax highlighting but at least its not bad.

File: docs/cynic_dpkg.md [hunk 14]
```diff
@@ -1,11 +1,6 @@
...
-
-
-*  All source modifications in /home/werdz/dpkg-whateveritis-nexentawhatever
-
-*  mkdir -p /var/lib/dpkg
-
-*  touch /var/lib/dpkg/status
-
-*  touch /usr/local/var/dpkg/status
+* All source modifications in /home/werdz/dpkg-whateveritis-nexentawhatever
```
Comment (VoyTechnology) [inline docs/cynic_dpkg.md]: Done.

File: docs/cynicdebian.md [hunk 15]
```diff
@@ -1,105 +1,124 @@
...
 ## Actually booting the damn install
 
-   * Get the CD, and put in in the disk.
-   * Break cynic to an ok prompt.
-   * setenv boot-device cdrom
-   * Yes, i said cdrom, not dvd rom.
-   * At this point, you're asking yourself why not just boot from the cd. You have to cold boot it, or it just doesn't work.
-   * Use the LOM to poweroff the machine, wait at least 10 seconds, then poweron.
-   * If it works then congratulations, you've passed the first hurdle. If you get an illegal instruction error you're doing it wrong.
+* Get the CD, and put in in the disk.
```
Comment (VoyTechnology) [inline docs/cynicdebian.md]: Realistically nobody ever will run through this checklist. We could do that, but very little need.

File: docs/deathray.md [hunk 16]
```diff
@@ -1,18 +1,15 @@
...
-
 The only services currently running on deathray are:
 
-
-*  
-
-*  
-
-*  
+*Edit(voy): It was an empty list...*
```
Comment (VoyTechnology) [inline docs/deathray.md]: I'll say keep it but never look at it again (aka. legacy)

File: docs/dovecot.md (hunk 1 above)
Comment (VoyTechnology) [inline docs/dovecot.md]: Acknowledged. Not part of this PR.

File: docs/dracaccess.md (hunk 2 above)
Comment (VoyTechnology) [inline docs/dracaccess.md]: TIL: Spaces and Tabs do not go well together.

File: docs/legacy/machines/murphy.md (hunk 3 above)
Comment (VoyTechnology) [inline docs/legacy/machines/murphy.md]: Acknowledged. Not part of this PR.

**Comment by VoyTechnology** [comment]
I don't mind continuing in this PR, but I think it might be better to cut it here and split the formatting into few PRs to avoid blocking on reviews and making it so I don't have to go back and forth between reviewed and not reviewed pages.

File: docs/dracaccess.md [hunk 17]
```diff
@@ -25,32 +25,27 @@ crap.
 
 ## DRAC network diagram:
 
- (136.206.15.8 )            [  192.168.1.x  ]
```
Comment (butlerx) [inline docs/dracaccess.md]: this is still needed

File: docs/imapupdatingcert.md [hunk 18]
```diff
@@ -1,36 +1,36 @@
...
-
 Important files (all on deathray)
 
-Cert File:              /etc/courier/imapd.pem\\
-Script:                 /local/admin/scripts/imap/updatecert.sh\\
-Expect File:    /local/admin/scripts/imap/script.exp
+```
+Cert File:			/etc/courier/imapd.pem
+Script:					/local/admin/scripts/imap/updatecert.sh
+Expect File:		/local/admin/scripts/imap/script.exp
```
Comment (butlerx) [inline docs/imapupdatingcert.md]: not a big thing but could you fix the tabbing

File: docs/imapupdatingcert.md [hunk 19]
```diff
@@ -1,36 +1,36 @@
...
+State or Province Name (full name) [Some-State]:	Leinster
+Locality Name (eg, city) []:											Dublin
+Organization Name (eg, company) [Internet Widgits Pty Ltf]:	RedBrick SSL IMAP
+Organizational Unit Name (eg, section) []:				Dublin City University Networking Society
+Common Name (eg, YOUR name) []:										imap.redbrick.dcu.ie
+Email Address []:																	admins@redbrick.dcu.ie
+A challenge password []:        										<leave blank>
+An optional company name []:												<leave blank>
+Enter PEM pass phrase:														imapcert
+```
```
Comment (butlerx) [inline docs/imapupdatingcert.md]: couldyou fix the tabbing here also ?

File: docs/index.md [hunk 20]
```diff
@@ -1,370 +1,311 @@
...
 #### Messaging
 
-*  [News Server (the boards)](news)
-
-*  [IRC](irc)
-
-*  <del>[jabber](jabber)</del>
+* [News Server (the boards)](news)
+* [IRC](irc)
+* <del>[jabber](jabber)</del>
```
Comment (butlerx) [inline docs/index.md]: i think you can remove anything with a del from index

Comment (butlerx) [inline docs/index.md]: never mind handled in #22

File: docs/committeechangeover.md (hunk 10 above)
Comment (CSmartt) [inline docs/committeechangeover.md]: List all lists? No idea.

File: docs/committeechangeover.md [hunk 21]
```diff
@@ -1,132 +1,133 @@
...
+	add: memberUid
+	memberUid: $new_admin_member1
+	memberUid: $new_admin_member2
+	memberUid: $new_admin_member3**
+	modifying entry "cn=root,ou=groups,o=redbrick"
+```
 
-Give each member a local account (in /etc/passwd) on each machine (TODO, document this, if necessary).
+* Give each new admin a local account (in `/etc/passwd`) on each machine
+(TODO, document this, if necessary).
```
Comment (CSmartt) [inline docs/committeechangeover.md]: I'll look after this section.

File: docs/committeechangeover.md (hunk 10 above)
Comment (butlerx) [inline docs/committeechangeover.md]: @VoyTechnology just remove it

File: docs/daniel.md [hunk 22]
```diff
@@ -1,35 +1,45 @@
...
+To create a VM use `/xen/scripts/vmcreate.sh`
 
-To create a VM use /xen/scripts/vmcreate.sh
+This Script takes information provided to it by you and uses this information to
+create a Xen config for a VM as well as a Disk Image. In order to keep things
+simple all VMs will be created as full hvm machines with an ISO attached for the
+OS required.
 
-This Script takes information provided to it by you and uses this information to create a Xen config for a VM as well as a Disk Image. In order to keep things simple all VMs will be created as full hvm machines with an ISO attached for the OS required.
+Symlings in /`xen/isos` need to point to the current ISO for a Distro, subsets
```
Comment (CSmartt) [inline docs/daniel.md]: symlinks*

File: docs/deathray.md (hunk 16 above)
Comment (CSmartt) [inline docs/deathray.md]: This particular page could become a wiki page on what Deathray used to do, and be removed from docs.

File: docs/dracaccess.md [hunk 23]
```diff
@@ -1,31 +1,30 @@
 # DRAC access
 
 ## Overview
-
 Our two poweredges have a DRAC, an embedded 'Dell Remote Access Controller' This allows us to:
```
Comment (CSmartt) [inline docs/dracaccess.md]: Need to check how many DRACs we currently have.

File: docs/services/irc.md [hunk 24]
```diff
@@ -1,48 +1,61 @@
 # RedBrick IRCd
-
 This runs on morpheus, irc.redbrick has it's own service address for this.
```
Comment (CSmartt) [inline docs/services/irc.md]: Paphos.
(Not part of this PR I know)

File: docs/ldap.md [hunk 25]
```diff
@@ -1,182 +1,205 @@
...
-Where: username is who you're editing\\
-loginShell is the attribute\\
-/usr/l... is the value
+> `/usr/l...` is the value
 
 Then run ldapmodify like this:
 
-    root@redbrick# ldapmodify -x -D cn=root,ou=ldap,o=redbrick -y /etc/ldap.secret -f modifyfile
+```
+root@redbrick# ldapmodify -x -D cn=root,ou=ldap,o=redbrick -y /etc/ldap.secret -f modifyfile
```
Comment (butlerx) [inline docs/ldap.md]: just for future docs i think it would be a good idea to do @redbrick reather then a machines hostname

File: docs/ldapsetup.md [hunk 26]
```diff
@@ -1,172 +1,274 @@
 # NB THIS IS A LEGACY PAGE KEPT FOR POSTERITY. THIS IS NOT AN ACCURATE REPRESENTATION OF THE Current Ldap Setup
 
 # Current Ldap Setup
-
-Ldap is running on Morpheus. The secondary ldap should be setup on deathray as soon as it is re-installed, and setup as a slave.
+Ldap is running on Morpheus. The secondary ldap should be setup on deathray as
```
Comment (butlerx) [inline docs/ldapsetup.md]: im just pointing out to @CSmartt this is in serious need of update

File: docs/committeechangeover.md (hunk 21 above)
Comment (VoyTechnology) [inline docs/committeechangeover.md]: I presume noop for me just a general comment.

File: docs/dracaccess.md (hunk 23 above)
Comment (VoyTechnology) [inline docs/dracaccess.md]: I presume this is also a noop for me.

**Comment by VoyTechnology** [comment]
@butlerx good to merge and continue with `M-Z` in another PR?
